apps/api/src/domain/models/bibliographic-reference/common/commands/register-digital-representation-of-bibiliographic-citation/register-digital-representation-of-bibliographic-citation.command.type-validation.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { plainToInstance } from 'class-transformer';
+import { CommandFSA } from '../../../../../../app/controllers/command/command-fsa/command-fsa.entity';
+import { InternalError } from '../../../../../../lib/errors/InternalError';
```

Comment:
I just recalled that we need to rename `BilbigraphicReference` to `BibliographicCitation` everywhere.

Reply:
[Here](https://www.pivotaltracker.com/story/show/186611085) is the story.